backend/dal/Services/IUserService.cs

```diff
@@ -14,6 +14,7 @@ public interface IUserService
         IEnumerable<int> GetAgencies(Guid userId);
         AccessRequest GetAccessRequest();
         AccessRequest GetAccessRequest(int id);
+        AccessRequest DeleteAccessRequest(AccessRequest accessRequest);
```

Comment:
Technically the service doesn't need to return the object deleted, only the controller does.  We'll add it to tech debt.